### ISSUE TITLE
First log messages are not shown

### ISSUE THREAD

**Issue by VSMent** [JetBrains/resharper-unity#1871]
First logs (from Start method) are not shown by rider, though they are present in unity.

Example code:
```
void Start(){
  Debug.Log("Start");
}
```
or 
```
bool first = true
void Update(){
  if(first){
    Debug.Log("Start");
    first = !first;
  }
}
```
Unity 2019.4.3f1 Personal
Rider 2020.2

**Comment by van800** [comment]
This is expected to be already fixed in 2020.3 EAP1 by https://github.com/JetBrains/resharper-unity/pull/1837

**Comment by VSMent** [comment]
@van800 Thanks)
